### ISSUE TITLE
"More tasks executed then were scheduled" error when using zone.js and jquery.layout.

### ISSUE THREAD

**Issue by NikGovorov** [angular/zone.js#290]
Please try to resize a window or hover on any resize pane:
http://plnkr.co/edit/coyaquV0zuwwVGLGAGUE?p=preview

**Comment by born2net** [comment]
+1 seeing same issues...

**Comment by born2net** [comment]
same issues in 0.12
```
More tasks executed then were scheduled.
```

**Comment by NikGovorov** [comment]
Confirming, same issue in  0.6.6: http://plnkr.co/edit/XLXdc1EoEzOE7O4VoPQO?p=preview

**Comment by NikGovorov** [comment]
@mhevery, Could you please re-open the issue?

**Comment by ghaiat** [comment]
+1
Same issue here

**Comment by fxck** [comment]
yes, it didn't help

**Comment by born2net** [comment]
this is a big showstopper for me...
I can't release ng 0.12 Kitchen sink: http://ng2.javascriptninja.io
due to this bug...

ng team, PLEASE remove the  Throw Exceptions in zone.js 

Regards,

Sean

**Comment by born2net** [comment]
please re-open

**Comment by arsenys** [comment]
I have a same bug, "More tasks executed then were scheduled." started to appear in my application that uses [Semantic-UI](http://semantic-ui.com/) for many components. Please re-open.

**Comment by MaikeMota** [comment]
Same problem here. The "More tasks executed then were scheduled" message always appear when has interation with semantic-ui elements.

![image](https://cloud.githubusercontent.com/assets/4526602/14023714/6c15ab80-f1c4-11e5-862c-4f645b6ffccd.png)

![image](https://cloud.githubusercontent.com/assets/4526602/14023734/870c8b48-f1c4-11e5-8217-e7ebe3e1dd01.png)

I'm using 2.0.0-beta.12 version.

**Comment by mhevery** [comment]
This will show up in beta.13

**Comment by mhevery** [comment]
OK, it seems like there as another issue with the same error message. Could someone please file a new issue with a plunker demonstrating the problem.

**Comment by frankwallis** [comment]
@mhevery - I think it is the same issue, this is using zone.js@0.6.6: 
http://plnkr.co/edit/XrjNURLeN5bhPrkXnohG?p=preview

**Comment by SelcukTekin** [comment]
Also want to leave my input here that we are experiencing the same problem which is incredibly easy to reproduce. Just create a function that creates a timeout every 10 seconds that triggers itself. 

Like is shown by @frankwallis example
It is a showstopper for us as well.

**Comment by cagataycivici** [comment]
Same problem here with the spinner component;

http://www.primefaces.org/primeng/#/slider

Fails on beta12, after update from beta9.

**Comment by NikGovorov** [comment]
Please see https://github.com/angular/zone.js/issues/302.

**Comment by 0xE282B0** [comment]
I get the "More tasks executed then were scheduled" on safari mobile.
I'm using ionic-angular@2.0.0-beta.3 and angular2@2.0.0-beta.12 with zone.js@0.6.8.

Can someone confirm that problem on safari mobile?

**Comment by ourway** [comment]
Remove angular2-polyfills and add zone and reflect-metadata with angular2-beta-13
```

...

 <script src="node_modules/reflect-metadata/Reflect.js"></script>
 <script src="node_modules/zone.js/dist/zone.js"></script>

...


```

**Comment by mjd5** [comment]
Removing angular2-polyfills and adding zone and reflect as per @ourway worked (with semantic ui) for me also, but not until I'd updated zone.js to 0.6.8 (on zone.js 0.6.6 I was still getting the "More tasks executed then were scheduled" errors as per #302)

**Comment by xx0r** [comment]
in my case upgrading the zone.js fixed the issue, but introduced a new one:

(angular2.0.0-beta.14, zone.js 0.6.8)

EXCEPTION: Error: Uncaught (in promise): false

![image](https://cloud.githubusercontent.com/assets/3314395/14389465/0889f6a2-fdb3-11e5-8b40-c228bdd3668d.png)

**Comment by kanchansingh1123** [comment]
Hey!

Awesome it is fixed for me after upgrading Angular to "2.0.0-beta.14" with zone "0.6.10" in my package.json and I run npm install.

**Comment by danielgerlag** [comment]
+1

**Comment by supamax** [comment]
Still get it on angular.beta.15

**Comment by candelibas** [comment]
+1

**Comment by mhevery** [comment]
Please file a separate issue with a plunker demonstrating the problem.

**Comment by Cajam** [comment]
+1
